lib/medtronic600/medtronic600Simulator.js

```diff
@@ -493,6 +359,12 @@ class Medtronic600Simulator {
             return false;
           }
         }
+      } else if (event.type === 'basal') {
+        // Filter out any basals with no duration. This can happen inside
+        // applySuspendedBasals(), when a basal is suppressed completely by a pump suspend
```

Comment:
Ah, I see further down that you may be using these events already? And then just filtering them out here at the end?